src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px $hug $hug;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+
+  transition: all 0.5s ease-in-out;
```

Comment:
It is recommended to specify the exact properties instead of all
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

Reply:
Ok, I got it)